src/app/state/auth/keycloak/keycloak.ts

```diff
@@ -0,0 +1,14 @@
+const settings = {
+  // TODO: Switch between production and acceptance realm
+  "realm": "datapunt-ad-acc",
```

Comment:
Can't we put this in the .env-file? So we can't forget to switch this.

Reply:
Yes, have done that. Please refresh.